### ISSUE TITLE
Breadcrumb: High contrast issues

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-core#628]
We have three reported issues around high contrast mode:

1. Arrows showing insertion cursor
 - When hovering over the nodes, you get the text insertion cursor
 - Should be the arrow cursor

2. Nodes should be hyperlink yellow
 - ![image](https://cloud.githubusercontent.com/assets/1382445/15807616/8c6d2cc8-2b17-11e6-8010-f73aeb5206ab.png)

3. Not showing hover in high contrast
 - No hover state
 - Even high contrast people need mouse feedback. Especially high-contrast people who use mouse. This should be the same as selection color.

**Comment by mikewheaton** [comment]
This issue was moved to OfficeDev/office-ui-fabric-js#56
